ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py

```diff
@@ -3087,3 +3087,52 @@ def test_tensorflow_Pack(
         values=x,
         axis=axis,
     )
+
+
+@st.composite
+def _pad_helper(draw, *, padding_types=None):
+    if padding_types is None:
```

Comment:
No need to have this implicitly set here, I'd suggest making making `padding_types` a required parameter or a fixed value, just to avoid the unnecessary overhead.